dashboard/db/migrate/20160219021859_add_submitted_to_user_levels.rb

```diff
@@ -0,0 +1,14 @@
+class AddSubmittedToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :submitted, :boolean
+
+    reversible do |change|
+      change.up do
+        execute('UPDATE user_levels SET submitted = true, best_result = 30 WHERE best_result = 1000')
```

Comment:
On my local machine it looks like the migration indeed leaves those timestamps intact...

before migration:

```
mysql> select * from user_levels where id = 35;
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+
| id | user_id | level_id | attempts | created_at          | updated_at          | best_result | script_id | level_source_id |
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+
| 35 |       1 |      357 |        3 | 2016-02-16 11:24:06 | 2016-02-19 01:20:09 |        1000 |        51 |            NULL |
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+
1 row in set (0.01 sec)

mysql> select * from user_levels where id = 36;
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+
| id | user_id | level_id | attempts | created_at          | updated_at          | best_result | script_id | level_source_id |
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+
| 36 |       1 |     3712 |       11 | 2016-02-16 11:24:52 | 2016-02-19 07:17:18 |          30 |        51 |            NULL |
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+
1 row in set (0.00 sec)
```

after migration:

```
mysql> select * from user_levels where id = 35;
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+-----------+
| id | user_id | level_id | attempts | created_at          | updated_at          | best_result | script_id | level_source_id | submitted |
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+-----------+
| 35 |       1 |      357 |        3 | 2016-02-16 11:24:06 | 2016-02-19 01:20:09 |          30 |        51 |            NULL |         1 |
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+-----------+
1 row in set (0.00 sec)

mysql> select * from user_levels where id = 36;
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+-----------+
| id | user_id | level_id | attempts | created_at          | updated_at          | best_result | script_id | level_source_id | submitted |
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+-----------+
| 36 |       1 |     3712 |       11 | 2016-02-16 11:24:52 | 2016-02-19 07:17:18 |          30 |        51 |            NULL |      NULL |
+----+---------+----------+----------+---------------------+---------------------+-------------+-----------+-----------------+-----------+
1 row in set (0.00 sec)
```